hack/jenkins/e2e-runner.sh

```diff
@@ -21,8 +21,53 @@ set -o nounset
 set -o pipefail
 set -o xtrace
 
+# include shell2junit library
+source <(curl -fsS --retry 3 'https://raw.githubusercontent.com/kubernetes/kubernetes/master/third_party/forked/shell2junit/sh2ju.sh')
```

Comment:
How is this running standalone? Don't you need to clone this repo to get access to the script in the first place?